x-pack/test/functional/apps/ml/stack_management_jobs/manage_spaces.ts

```diff
@@ -179,7 +179,12 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.navigation.navigateToStackManagementJobsListPage();
 
           // AD
-          await ml.jobTable.filterWithSearchString(testData.adJobId, 1, 'stackMgmtJobList');
+          await ml.stackManagementJobs.filterTableWithSearchString(
+            'anomaly-detector',
+            testData.adJobId
+          );
+
+          // await ml.jobTable.filterWithSearchString(testData.adJobId, 1, 'stackMgmtJobList');
```

Comment:
Could remove this left-over line

Reply:
Updated in https://github.com/elastic/kibana/pull/134893/commits/fa39fc25ce8bd674304641dfade7854216def157